JournalMonitor/JournalMonitor.cs

```diff
@@ -153,7 +153,8 @@ public static List<Event> ParseJournalEntry(string line)
                                 Economy economy = Economy.FromEDName(getString(data, "SystemEconomy"));
                                 Government government = Government.FromEDName(getString(data, "SystemGovernment"));
                                 SecurityLevel security = SecurityLevel.FromEDName(getString(data, "SystemSecurity"));
-                                long? population = getOptionalLong(data, "Population");
+                                data.TryGetValue("Population", out val);
```

Comment:
This indicates a problem with `getOptionalLong()`. Why not fix that instead? (Genuine question).